components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -1996,7 +1996,8 @@ private OMElement queryBetweenTwoDaysForFaulty(String columnFamily, String fromD
         }
     }
 
-    private OMElement queryToGetAPIUsageByResourcePath(String columnFamily, String fromDate, String toDate)
+    private List<APIUsageByResourcePath> queryToGetAPIUsageByResourcePath(String tableName, String fromDate,
```

Comment:
rename method as "getAPIUsageByResourcePathData" and fixed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621